README.md

```
@@ -230,21 +230,25 @@ This solution consists of 6 parts that are being developed with Jetbrains Rider
 * [DDD and Hexagonal Architecture](https://vaadin.com/blog/ddd-part-3-domain-driven-design-and-the-hexagonal-architecture)
 
 ## About Management
+
 ### Security Management
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)